.subjects/STUD_PART/exam_06/0/mini_serv/catchmsg.sh

```diff
@@ -2,7 +2,6 @@
 
 # close fd before init
 exec 6<&-
-nc localhost $1
```

Comment:
This line was connecting a new client before the program could read it, so the given output started on client 0 and the tested program started on client 1, causing a failure when the program was correct.

We can remove this line, as it behaves properly without it, or we could change the given test output to start on 1 instead.